### PR TITLE
Zendesk chat v2 documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Zendesk v2 source documentation

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Comment by ameypv-rudder** [comment]
@am6010 @nidhilashkari17 have a couple of queries related to this PR. Please refer to [this thread](https://rudderlabs.slack.com/archives/C01DY56B1U1/p1664516354951509) for more info.

File: docs/sources/extract/zendesk-chat-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+### Connection settings
+
+To set up Zendesk Chat as a Cloud Extract source, you need to configure the following settings:
+
+<img src="../../assets/cloud-extract-sources/zendesk-chat-connection-settings.png" alt="Zendesk Chat connection settings" />
+
+- **Start Date**: Choose the start date from which you want RudderStack to ingest the Zendesk Chat data. RudderStack will not replicate any data before this date.
+
+<div class="warningBlock">
+If you do not specify the start date, RudderStack will replicate all your data by default.
```
Comment (am6010) [inline docs/sources/extract/zendesk-chat-v2.mdx:40]: If they don't provide a starting date, they won't be able to press continue and create the source.

File: docs/sources/extract/zendesk-chat-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+</div>
+
+- **Subdomain**: Enter the subdomain from your Zendesk account's URL. For example, if your Zendesk account URL is `sample.zendesk.com`, the subdomain would be `sample`.
```
Comment (am6010) [inline docs/sources/extract/zendesk-chat-v2.mdx:43]: It is **mandatory** to provide a subdomain if their zendesk url follows this format `<subdomain>.zendesk.com`. 
If they have only a zendesk chat account and the url doesn't follow this format they may connect without a subdomain.

**Review by am6010 — COMMENTED** [review]
Looks very good! I added some small comments

File: docs/sources/extract/zendesk-chat-v2.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/sources/extract/zendesk-chat-v2.mdx:40]:
```suggestion
You will not be able to proceed if you do not specify the start date.
```

File: docs/sources/extract/zendesk-chat-v2.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/sources/extract/zendesk-chat-v2.mdx:43]: Thanks @am6010 for these useful inputs! I've incorporated these pointers in the doc.

**Review by am6010 — APPROVED** [review]
Thank you for the changes!

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**